deployments/ansible/roles/collector/README.md

```diff
@@ -135,6 +135,22 @@ $> ansible-playbook playbook.yaml -e start_service=false
 
 - `start_service`: Whether to restart the services installed by the playbook. (**default:** true)
 
+- `splunk_otel_collector_custom_env_vars`: Hash of custom environment variables from the
```

Comment:
nit 
```suggestion
- `splunk_otel_collector_custom_env_vars`: Dictionary of additional environment variables from the
```

Reply:
@rmfitzpatrick Do you think I should also rename `splunk_otel_collector_custom_env_vars` to something like `splunk_otel_collector_additional_env_vars`?  Or maybe just `splunk_otel_collector_env_vars`?

Reply:
I was thinking something similar but you documented a use case well enough to convince me not to ask :) . `additional_` is more accurate?